contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -160,12 +160,15 @@ func (t *T) FillFromDetails(details *mesos.Offer) error {
 		panic("offer details are nil")
 	}
 
-	log.V(3).Infof("Recording offer(s) %v against pod %v", details.Id, t.Pod.Name)
+	cpu := t.computeResourceLimit("cpu", DefaultContainerCpus)
+	mem := t.computeResourceLimit("memory", DefaultContainerMem)
+
+	log.V(3).Infof("Recording offer(s) %v against pod %v: cpu: %f, mem: %f", details.Id, t.Pod.Name, cpu, mem)
```

Comment:
if we use k8s resource.Quantity it has nice output funcs for printing quants in a pretty human readable way.